app/controlplane/api/controlplane/v1/workflow_run.proto

```diff
@@ -77,6 +77,7 @@ message AttestationServiceGetPolicyResponse {
 message RemotePolicyReference {
   string url = 1;
   string digest = 2;
+  string org_name = 3;
```

Comment:
If I'm not wrong, the org is already in the URL in the form of a URL parameter `org=my-org`. Is this what we need?

Reply:
That's true, changed, thanks!